File: problemas/ad-hoc/2381/2381.cpp
#include <algorithm>
#include <iostream>
#include <vector>
using namespace std;

int main() {
	int N, K;
	vector<string> alunos;
	
	cin >> N >> K;
	alunos.assign(N, "");
	for(int i = 0; i < N; ++i){
		cin >> alunos[i];
	}
	
	sort(alunos.begin(), alunos.end());
	
	cout << alunos[K - 1] << endl;
	
	return 0;
}

File: problemas/ad-hoc/2381/2381.r
input <- file('stdin', 'r')

entrada <- as.integer(strsplit(readLines(input, n = 1), ' ')[[1]])
n <- entrada[1]
k <- entrada[2]

alunos <- c()
for(i in 1:n){
  alunos <- c(alunos, readLines(input, n = 1))
}
alunos <- sort(alunos)
write(paste(alunos[k]), '')